src/frontend/alerts.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('vex-js/css/vex.css');
+require('vex-js/css/vex-theme-default.css');
+
+const vex  = require('vex-js/js/vex.js');
+vex.dialog = require('vex-js/js/vex.dialog.js');
+vex.defaultOptions.className = 'vex-theme-default';
+
+module.exports = {
+    confirmate: function(message, callback) {
```

Comment:
from the one side -- I fully agree, but from another it will be like default window.confirm method and it's a little confused, on your wish